feat: Open configuration popups on the wallet page

liordino opened the pull request:
feat: open configuration popup when selecting a non-configured third
party service on the wallets page

liordino committed:
feat: open configuration popup when selecting a non-configured third
party service on the wallets page

liordino commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Open configuration popups on the wallet page](https://trello.com/c/QovhmttA/239-open-configuration-popups-on-the-wallet-page)

RafaelTaranto commented on new-lamassu-admin/src/pages/Wallet/Wallet.js:
```diff
@@ -17,6 +20,13 @@ const SAVE_CONFIG = gql`
   }
 `
 
+// TODO: should I use the save config and send the config as it is when saving an account?
```
Nope, this is fine. You can remove this comment.

RafaelTaranto commented on new-lamassu-admin/src/pages/Wallet/helper.js:
```diff
@@ -53,7 +60,8 @@ const getElements = (cryptoCurrencies, accounts, wizard = false) => {
         getLabel: R.path(['display']),
-        optionsLimit: null
+        optionsLimit: null,
+        onChange
```
Ticker's don't use third party services.

liordino committed:
fix: removed unnecessary comment

fix: removed the onChange event from the ticker Autocomplete, as it
don't use third party services
